config/gulp/file-upload.js

```diff
@@ -1,69 +1,51 @@
-require("dotenv").config();
-const { series, src } = require("gulp"),
-  vinylPaths = require("vinyl-paths"),
-  del = require("del"),
-  s3config = {
-    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+require("dotenv").config(); 
+
+const gulp = require("gulp");
+const awspublish = require('gulp-awspublish');
+const del = require("del");
+const vinylPaths = require("vinyl-paths");  
+
+// Create a new publisher using S3 options
+const publisher = awspublish.create({
+  region: 'us-east-1', // Change to your AWS region
+  params: {
+    Bucket: 'digitalgov', // Change to your bucket name
   },
-  s3 = require("gulp-s3-upload")(s3config);
-
-/**
- * Uploads images in /processed folder to digitalgov s3 bucket
- * TODO: Refactor both functions into one
- * https://github.com/uswds/uswds-compile/blob/223fe67c335135c55cf1be9dead44e6d363e219d/gulpfile.js#L204-L232
- */
-
-function uploadImage() {
-  console.log("starting image upload");
-
-  return src("content/uploads/_working-images/processed/*")
-    .pipe(
-      s3(
-        {
-          Bucket: "digitalgov",
-          ACL: "public-read",
-        },
-        {
-          maxRetries: 5,
-        }
-      )
-    )
-    .pipe(vinylPaths(del));
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+ 
+function uploadImage() { 
+  return gulp.src("content/uploads/_working-images/processed/*")
+  .pipe(publisher.publish())  
+  .pipe(awspublish.reporter({
+    states: ['create', 'update', 'delete']
+  }))
+  .pipe(vinylPaths(del))
+  .on('error', function(err) {
+    console.error('Image upload failed:', err);
+  });
 }
 
-/**
- * Uploads files in /to-process folder to s3 static file bucket
- * TODO: Refactor both functions into one
- */
-
-function uploadFile() {
-  console.log("starting file upload");
-
-  return src("content/uploads/_working-files/to-process/*")
-    .pipe(
-      s3(
-        {
-          Bucket: "digitalgov/static",
-          ACL: "public-read",
-        },
-        {
-          maxRetries: 5,
-        }
-      )
-    )
-    .pipe(vinylPaths(del));
+function uploadFile() { 
+  publisher.config.params.Bucket = "digitalgov/static";
+  return gulp.src("content/uploads/_working-files/to-process/*")
+    .pipe(publisher.publish())
+    .pipe(awspublish.reporter({
+      states: ['create', 'update', 'delete']
+    }))
+    .pipe(vinylPaths(del))
+    .on('error', function(err) {
+      console.error('File upload failed:', err);
+    }); 
```

Comment:
I still get `InvalidBucketName: Bucket name shouldn't contain '/', received 'digitalgov/static'`. The solution looks to be renaming the file to include `static/filename.pptx` before uploading.

We should remove line 31 and use [gulp-rename](https://www.npmjs.com/package/gulp-rename#usage) to include the path in the filename.